### ISSUE TITLE
[Bug]: Freezing, cramping, unresponsive issue when opening tab URL in sidebar

### ISSUE THREAD

**Issue by universish** [Floorp-Projects/Floorp#1494]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have checked with a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked that this problem is not occurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

I added some website URLs to the sidebar. for each URL I created a separate container and set it to work in that container. when I click on the website icon in the sidebar, it works in that container. When it starts to run, it freezes excessively, it stutters excessively, it consumes CPU all the time, finally it becomes unresponsive. Then the unresponsive, wait for it to respond or close option window pops up. Even if you click wait for it to respond, this window pops up again. It keeps repeating. The browser becomes inoperable.
OS: fedora 40

### Reproduction Steps

-

### Expected Behavior

-

### Actual Behavior

-

### Screenshots

-

### Browser Version

11.19.1

### Operating System

fedora 40

**Comment by surapunoyousei** [comment]
Please present the video.

**Comment by universish** [comment]
I was having this problem with the floorp flathub version. Yesterday I deleted flathub and started using portable. The vertical tab setting is on. In the vertical tab, in the settings; “Color the Tab Bar with the colors of the tab icon of the open site” is on. It seems like the portable version doesn't have this problem. 
Since I deleted the floorp flathub package, unfortunately I can't show or video this problem.
